src/attributes.rst

```diff
@@ -1452,15 +1452,15 @@ Macros Attributes
 
 .. _fls_qyudjGHZfyJH:
 
-Attribute ``collapsable_debuginfo``
+Attribute ``collapse_debuginfo``
 ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
```

Comment:
Nit: the length of the header underline is not the same. I *think* it might cause Sphinx build to fail.